### PR TITLE
Add case sensitivity option to search controls; Fix incorrect logger debug message in `StatsCollectionName`.

### PR DESCRIPTION
**Pull request by junhaoliao**
# References
1. Search option `ignore_case` was recently added for `clp-s` (#272). 

# Description
1. Add case sensitivity option to search controls.
2. Fix incorrect logger debug message in `StatsCollectionName`.

# Validation performed
1. Loaded Web UI in a browser.
2. In the search controls, observed that `Case sensitivity: Insensitive` as the default option. 
3. With `hadoop-24hrs/logs/i-00c90a0f/` compressed, in the Web UI searched with keyword "downloading", `Time Range` as `All Time`, and `Case sensitivity: Insensitive`, and observed "Results count: 376".
4. In the search controls, changed `Case sensitivity` to be `Sensitive` and restarted the search. Observed "Results count: 0".

### CODE REVIEW

**Comment by junhaoliao** [comment]
Does it make more sense to place "Insensitive" (the default setting) on the left instead?

image

**Comment by kirkrodrigues** [comment]
> Would it make more sense to place "Insensitive" (the default setting) on the left instead?

Sounds good.

**Review by kirkrodrigues — CHANGES_REQUESTED** [review]
Can we set a max-width (or some other solution) for the column labels so that they don't expand too much like this?

image

**Comment by junhaoliao** [comment]
> Can we set a max-width (or some other solution) for the column labels so that they don't expand too much like this?
> 
> image

It seems the gap was due to missing sizes of different breakpoints (e.g. xs, sm, md, lg, xl, xxl). In the latest attempt, I still cannot find the best numbers to avoid the labels being broken into two lines while keeping the label not too wide. Maybe that's a compromise we have to make if we want the label width to scale with page width. (Let me know if the scaling behaviour was not intended / that the label being broken into two lines when displayed in an odd resolution is not acceptable. Then your suggestion to simply set a `min-width` and `width` is probably the best way to go with. )

**Comment by kirkrodrigues** [comment]
> Maybe that's a compromise we have to make if we want the label width to scale with page width. (Let me know if the scaling behaviour was not intended / that the label being broken into two lines when displayed in an odd resolution is not acceptable. Then your suggestion to simply set a `min-width` and `width` is probably the best way to go with. )

Yeah, I'd prefer if the label width didn't scale (intuitively, I don't the user would expect it).

**Comment by junhaoliao** [comment]
Kept the Bootstrap Grid system and only limited the container size :)
image
image
image
image

**Review by kirkrodrigues — CHANGES_REQUESTED** [review]
Sorry, not to beat a dead horse, but the spacing still seems a bit too wide:

image

File: components/webui/imports/ui/SearchView/SearchControls.jsx [hunk 1]
```diff
@@ -42,27 +42,49 @@ const SearchControlsDatePicker = (props) => (<DatePicker
...
-    xs={"auto"}
-    className="search-filter-control-label"
-/>);
+const SearchControlsFilterLabel = (props) => (
+    <Form.Label
+        {...props}
+        className={"search-filter-control-label"}
+        column={"sm"}
+        sm={2}
+        llg={1}/>
```
[the remark this answers is not in the document]
Comment (junhaoliao) [reply components/webui/imports/ui/SearchView/SearchControls.jsx]: I meant to configure `xxl`. 🤦

Still, in the latest commit I got it fixed by specifying a `min-width` on the label where we can keep the Grid system with the label text broken into two lines. 

image

File: components/webui/imports/ui/SearchView/SearchControls.jsx [hunk 2]
```diff
@@ -93,6 +115,16 @@ const SearchFilterControlsDrawer = ({
         setTimeRange(timeRange);
     };
 
+    /**
+     * Handles case sensitivity change.
+     *
+     * @param {InputEvent} event
+     * @returns {void}
```
[the remark this answers is not in the document]
Comment (junhaoliao) [reply components/webui/imports/ui/SearchView/SearchControls.jsx:121]: Feel free to object - 

I think it's better to explicitly state the return type for every function, and we can rely on `valid-jsdoc` to catch the ones that are not supposed to return `void` but we do not have the `@returns` specified in the JSDoc.

[the remark this answers is not in the document]
Comment (kirkrodrigues) [reply components/webui/imports/ui/SearchView/SearchControls.jsx:121]: Does valid-jsdoc warn if we don't have `@returns {void}`?

**Review by kirkrodrigues — DISMISSED** [review]
Commit msg: `webui: Add case-sensitivity search control; Fix incorrect debug log. (#280)`?

File: components/webui/imports/ui/SearchView/SearchControls.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (kirkrodrigues) [reply components/webui/imports/ui/SearchView/SearchControls.jsx:121]: Haven't tried it, but looking at the [docs](https://eslint.org/docs/latest/rules/valid-jsdoc#options), it looks like we can specify `requireReturn: false` to have `valid-jsdoc` only warn for methods that actually return something.

[the remark this answers is not in the document]
Comment (junhaoliao) [reply components/webui/imports/ui/SearchView/SearchControls.jsx:121]: Let me remove the `@returns` to keep the docs less verbose then.

**Review by kirkrodrigues — APPROVED** [review]

**Pull request merged**